include/seeds.gratitude.hpp

```diff
@@ -120,8 +149,25 @@ CONTRACT gratitude : public contract {
     const name gratzgen_res = "gratz1.gen"_n; // Gratitude generated per cycle for residents
     const name gratzgen_cit = "gratz2.gen"_n; // Gratitude generated per cycle for citizens
     const name gratz_acks = "gratz.acks"_n; // Gratitude generated per cycle for citizens
+    const name gratz_potkp = "gratz.potkp"_n; // Percentil of the pot to keep each round
```

Comment:
if this is configurable, there needs to be a unit test that tests for this being != 0

Reply:
Added the test, thanks!